crates/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -324,22 +320,28 @@ impl<D: ::ndarray::Dimension> TryFrom<::ndarray::Array<half::f16, D>> for Tensor
                 name: None,
             })
             .collect();
-        value
-            .is_standard_layout()
-            .then(|| TensorData {
+        match value.is_standard_layout() {
+            true => Ok(TensorData {
                 shape,
-                data: TensorBuffer::F16(
+                buffer: TensorBuffer::F16(
                     bytemuck::cast_slice(value.into_raw_vec().as_slice())
                         .to_vec()
                         .into(),
                 ),
-                meaning: TensorDataMeaning::Unknown,
-                meter: None,
-            })
-            .ok_or(TensorCastError::NotContiguousStdOrder)
+            }),
+            false => Ok(TensorData {
+                shape,
+                buffer: TensorBuffer::F16(
+                    value
+                        .iter()
+                        .map(|f| arrow2::types::f16::from_bits(f.to_bits()))
+                        .collect::<Vec<_>>()
+                        .into(),
+                ),
+            }),
+        }
```

Comment:
a `match { true, false }` is a bit weird I have to say :sweat_smile:

Reply:
:laughing: fair